### PR TITLE
feat: some general additions

### PR DESCRIPTION
**Pull request by loujaybee**
Wooooo, this is awesome @rishabkumar7 + @madebygps !!

I really like the timeline behind it, makes it very explicit and clear for a beginner. 

And I like how each phase has the same / similar structure in the docs, makes it easy to follow.

Just a start, I'll come back and iteratively add to it over time...

### CODE REVIEW

File: phase2/README.md [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
 
-## Things you should be able familiar with at the end of this phase
+## Things you should be familiar with at the end of this phase
 
-- TODO
+- Variables
+- Functions (defining them, function arguments)
+- Basic data types and their uses (strings, booleans etc)
+- Iteration (for loops, etc)
+- Unit testing (at least an apprecation / understanding of it)
```
Comment (loujaybee) [inline phase2/README.md:36]: Some additions here.

File: phase3/README.md [hunk 2]
```diff
@@ -25,62 +25,56 @@ You can skip the entry level one a take more time studying for the associate, or
...
-### AWS
-
-| Level      | Title    | Notes     |
-| :------------- | :---------- | ----------- |
-| Cloud Practitioner | [freeCodeCamp](https://youtu.be/3hLmDS179YE)   | Andrew Brown has great courses on freeCodeCamp's YouTube channel  |
-| Solutions Architect Associate   | [freeCodeCamp](https://youtu.be/Ia-UEYYR44s)| Andrew Brown has great courses on freeCodeCamp's YouTube channel |
-| Developer Associate  | [freeCodeCamp](https://youtu.be/RrKRN9zRBWs) | Andrew Brown has great courses on freeCodeCamp's Youtube Channel |
-| SysOps Administrator | [freeCodeCamp](https://youtu.be/KX_AfyrhlgQ)   | Andrew Brown has great courses on freeCodeCamp's Youtube Channel |
-| AWS | [Rishab's Cloud Notes](https://notes.rishab.cloud) | I don't have recent first hand experience with any AWS content besides A Cloud Guru, but my good friend [Rishab](https://twitter.com/rishabk7) does, checkout his very detailed exam guides :)
-
```
Comment (loujaybee) [inline phase3/README.md:50]: Not deleted, just moved down, to bring "how to approach learning" before the resources.

File: phase3/README.md [hunk 3]
```diff
@@ -25,62 +25,56 @@ You can skip the entry level one a take more time studying for the associate, or
...
-## Things you should be able familiar with at the end of this phase
+## Things you should be familiar with at the end of this phase
 
-- TODO
+- Working knowledge of how access permissions are granted for users and resources (AWS: IAM)
+- Working knowledge of the main storage services, and good hands-on experience with at least one (AWS: S3, RDS & DynamoDB)
+- Working knowledge of the compute services, good hands-on experience with at least one (AWS: EC2, AWS Lambda & ECS)
+- An appreciation for secrets management (AWS: KMS & Parameter Store)
 
 ## Certifications you might want to look into
```
Comment (loujaybee) [inline phase3/README.md:79]: Might be worth removing this section since there's "resources" above? Or merge into one?

File: phase3/README.md [hunk 4]
```diff
@@ -25,62 +25,56 @@ You can skip the entry level one a take more time studying for the associate, or
...
+ | Deploy a [ghost blog](https://ghost.org/docs/install/)                  | Blogs are great for you to document your learning, here's [mine](https:madebygps.com), deployed on Azure :) |
 
-## Things you should be able familiar with at the end of this phase
+## Things you should be familiar with at the end of this phase
 
-- TODO
+- Working knowledge of how access permissions are granted for users and resources (AWS: IAM)
+- Working knowledge of the main storage services, and good hands-on experience with at least one (AWS: S3, RDS & DynamoDB)
+- Working knowledge of the compute services, good hands-on experience with at least one (AWS: EC2, AWS Lambda & ECS)
+- An appreciation for secrets management (AWS: KMS & Parameter Store)
```
Comment (loujaybee) [inline phase3/README.md:77]: Some additions here, my Azure is terrible, so I'd need to look up the equivalents!

File: phase4/README.md [hunk 5]
```diff
@@ -6,43 +6,49 @@ Final phase. First off, read some stuff about what DevOps is. Each cloud platfor
...
+I also think it's worth reading [The Phoenix Project](https://itrevolution.com/the-phoenix-project/) it's a pretty fun read because it's a fictional story but it explains the importance of DevOps. But if you'd prefer something more like a textbook with similiar ideas, check out [The DevOps Handbook](https://itrevolution.com/the-devops-handbook/).
 
 ## DevOps practices
 
 - Continuous integration and continuous delivery (CI/CD)
 - Version Control
 - Infrastructure as Code
 - Configuration management
-- Continuous monitoring
+- Monitoring (logs, metrics, traces)
```
Comment (loujaybee) [inline phase4/README.md:17]: I'm not aware of "continuous monitoring" as a phrase? (Maybe it is and I missed the boat!). Suggestion to add data types here to make it more explicit what we mean by "monitoring" to a newbie!

File: phase4/README.md [hunk 6]
```diff
@@ -6,43 +6,49 @@ Final phase. First off, read some stuff about what DevOps is. Each cloud platfor
...
+## Things you should be familiar with at the end of this phase
 
-- TODO
+- Working knowledge of at least one continuous integration tool (Github Actions, Jenkins, etc)
+- An understanding of infrastructure as code (why you'd use it, etc)
+- An understanding of containerisation (why/when you'd need it / use containers)
+- An understanding of the practice of Continuous Integration (why it's needed)
+- An appreciation for the ideas behind: "immutable infrastructure"
+- An appreciation for monitoring and the basic ideas of logs, metrics, traces
+- An appreciation for "declarative and imperative infrastructure as code"
```
Comment (loujaybee) [inline phase4/README.md:46]: Some additions, some opinions might be debatable, so I'd be open to suggestions/modifications. I tried to make the language clear on what's a must-have vs the nice-to-haves / you should know about but maybe not have load of knowledge on...

File: phase2/README.md [hunk 7]
```diff
@@ -1,40 +1,44 @@
...
+| Order | Title                                                                        | Notes                                                                                       |
+| :---- | :--------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------- |
+| 1     | [Python Crash Course](https://ehmatthes.github.io/pcc/)                      | Another fun book to follow, also there is an Appendix in there on Git which is great!       |
+| 1     | [Automate the boring stuff with Python](https://automatetheboringstuff.com/) | I haven't fully finished this book, but it can be used to learn Python too                  |
+| 1     | [FreeCodeCamp Learn Python](https://www.youtube.com/watch?v=rfscVS0vtbw)     | One of the many amazing resources provided by [FreeCodeCamp](https://www.freecodecamp.org/) |
+| 2     | [Git tutorial for beginners](https://www.youtube.com/watch?v=8JJ101D3knE)    | You'll need Git, learn it!                                                                  |
```
Comment (loujaybee) [inline phase2/README.md:14]: My IDE reformatted the tables 😝

**Comment by madebygps** [comment]
thanks Lou! all this looks great.

**Pull request merged**